.travis.yml

```diff
@@ -24,6 +24,8 @@ addons:
   postgresql: '9.4'
 before_install:
 - source ${TRAVIS_BUILD_DIR}/tools/ci/before_install.sh
+install:
+- bundle update
```

Comment:
@bdunne Let's discuss...I don't think this is sufficient for bundle caching, but it should get the branch green, hopefully.